setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="celobox",
+    version="0.1a",
+    packages=find_packages(),
```

Comment:
I think `find_packages()` is a bit of overkill for something this simple. There's just one package, `celobox`, and even then it only really offers the CLI interface; it's not (currently) setup to provide any sort of useful API for other python packages to consume.